.github/workflows/radar-pre-release-actions.yml

```diff
@@ -0,0 +1,19 @@
+name: Publish beta package to npmjs
+on:
+  release:
+    types: [ prereleased ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # Setup .npmrc file to publish to npm
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+      - run: npm ci --force
```

Comment:
Do you know if `--force` is required here? We should probably remove it unless it's needed (it's a holdover from an old version of rollup, I need to remove it from the JS SDK as well)

Reply:
Oh, no I don't. I added it after seeing it in the web sdk. I'll remove.